templates/solutions/open-source-security/cyber-resilience-act/index.html

```diff
@@ -0,0 +1,591 @@
+{% extends "templates/base.html" %}
+
+{% from "_macros/vf_tiered-list.jinja" import vf_tiered_list %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/13VaiP6GGkGB4mE01bIqJ3cg8MfQXulNs9m3cmYb20HQ
+{% endblock meta_copydoc %}
+
+{% block title %}Understand the Cyber Resilience Act (CRA) and its requirements{% endblock %}
+
+{% block meta_description %}
+  Discover what you need to know about the EU Cyber Resilience Act requirements, and get clear recommendations for compliance. Understand how Canonical can help.
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block outer_content %}
+
+  {% block content %}
+
+    <section class="p-section--hero">
+      <div class="row--50-50-on-large">
+        <div class="col">
+          <h1 class="u-no-margin--bottom">Meet your Cyber Resilience Act requirements with Canonical</h1>
+        </div>
+        <div class="col">
+          <div class="p-section--shallow u-hide--large"></div>
+          <div class="p-section--shallow">
+            <p>
+              Build your products and device software with Ubuntu and Canonical's trusted open source portfolio. Fast-track your pathway to CRA compliance.
+            </p>
+          </div>
+          <div class="p-cta-block">
+            <a href="#"
```

Comment:
There should be a fallback URL in case user doesn't have JS enabled. I see that it is not included in the copy doc. Could you please check with Gloria or UX?

Reply:
I think we can leave the fallback for now because the site needs to be up before the holidays